src/api/db/data/20201112083506_remove_role_user.rb

```diff
@@ -0,0 +1,14 @@
+class RemoveRoleUser < ActiveRecord::Migration[6.0]
+  def up
+    role = Role.find_by(title: 'User')
```

Comment:
in Rails 6 you can simplify it with a `Role.destroy_by(title: 'User')` <3

Reply:
Cool, I didn't know about that method.

Reply:
Please have another look :)